CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 ## Contributing Guidelines
-This repo contains an alpha release of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+This repo contains an alpha release of some of the foundational aspects of KubeFed. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
 
 ## Sign the CLA
 Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more information
 
-## Learn About Kubernetes Cluster Federation
-If you want to learn about Kubernetes Cluster Federation, please read through the doc:
-https://github.com/kubernetes-sigs/kubefed#kubernetes-cluster-federation
+## Learn About KubeFed
+If you want to learn about KubeFed, please read through the doc:
+https://github.com/kubernetes-sigs/kubefed#kubefed
 
-To learn how to use Kubernetes Cluster Federation, please read through the User Guide:
+To learn how to use KubeFed, please read through the User Guide:
 https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md
 
-If you are interested in contributing to Kubernetes Cluster Federation project, please read through the Development Guide:
+If you are interested in contributing to KubeFed, please read through the Development Guide:
```

Comment:
Please revert

Reply:
Done, but I removed "project" in the latest version. If we wanted to keep project, I think I'd need to add "the" to make it "the Kubernetes Cluster Federation project".